tests/listsAndDropDowns.spec.ts

```diff
@@ -37,21 +37,20 @@ test.describe('Lists and Drop Downs', () => {
         const editButton = page.locator('.dl-horizontal', { hasText: "Rosy" }).getByRole('button', { name: 'Edit Pet' })
         await editButton.click()
         await expect(petNameInputField).toHaveValue('Rosy')
-        await expect(petTypeInputField).toHaveValue('dog')
+        await expect(petTypeField).toHaveValue('dog')
         await petTypeInputField.click()
         await petTypeInputField.selectOption('bird')
+        await expect(petTypeField).toHaveValue('bird')
         await expect(petTypeInputField).toHaveValue('bird')
         const updateButton = page.getByRole('button', { name: 'Update Pet' })
         await updateButton.click()
-        await expect(petNameInputField).toHaveValue('Rosy')
-        await expect(petTypeInputField).toHaveValue('bird')
+        await expect(page.locator('.dl-horizontal', { hasText: "Rosy" })).toContainText('bird')
         await editButton.click()
         await petTypeInputField.selectOption('dog')
         await expect(petTypeInputField).toHaveValue('dog');
+        await expect(petTypeField).toHaveValue('dog');
         await updateButton.click()
-        await expect(petNameInputField).toHaveValue('Rosy')
-        await expect(petTypeInputField).toHaveValue('dog')
-
+        await expect(page.locator('.dl-horizontal', { hasText: "Rosy" })).toContainText('dog')
```

Comment:
The locator `page.locator('.dl-horizontal', { hasText: "Rosy" })` was copy/pasted in this test 3 times, so it's definitely a good candidate to extract to the constant.